packages/fxa-payments-server/src/routes/Product/SubscriptionRedirect/Redirect123DonePro.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect } from 'react';
+import { SubscriptionRedirectProps } from './index';
+
+export const RedirectDefault = ({
+  navigateToUrl,
+  plan: {
+    plan_name
+  }
+}: SubscriptionRedirectProps) => {
+  const redirectUrl = 'http://127.0.0.1:8080/'; 
```

Comment:
Is this template always going to be the same except for redirectUrl?

Reply:
Not necessarily, this could be arbitrary marketing content

Reply:
Oh, but I think I'm more thinking this about the PlanDetails family of components. Yeah, this *might* mainly be a container for the redirect URL, at the end of the day.